src/Sass/Base/_base.scss

```diff
@@ -0,0 +1,20 @@
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+}
+
+html  {
+  box-sizing: border-box;
+  font-size: 62.5%; // 1rem = 10px 10px/16px = 62.5%
```

Comment:
Why use a percentage instead of pixels?

Reply:
It converts the font size to nicely use rem unit of 10 on all text, this also helps with a user resizing a window, because the user on zoom +/- deals with the pixel and not the rem, the ratio we set in styling wont change. It will still alow a user to zoom and change the font-sizes but help prevent bugs.

Reply:
I've heard a lot of people talk about the benefits of rem and I've never seen any of them. Would you show me an example today?